Compilation error with `#include <KokkosBatched_Getrs.hpp>`

When including the header `KokkosBatched_Getrs.hpp` I get the following error:
```
/Users/runner/work/ddc/ddc/opt/kokkos-kernels/include/KokkosBatched_Getrs_Serial_Internal.hpp:23:18: error: no member named 'SerialLaswp' in namespace 'KokkosBatched'
   23 |   KokkosBatched::SerialLaswp<Direct::Forward>::invoke(piv, b);
      |   ~~~~~~~~~~~~~~~^
```

See https://github.com/CExA-project/ddc/actions/runs/22639370093/job/65610889320?pr=1075 for the complete log.

I think the file https://github.com/kokkos/kokkos-kernels/blob/develop/batched/dense/impl/KokkosBatched_Getrs_Serial_Internal.hpp is missing at least these two include directives:
```
#include <KokkosBatched_Laswp.hpp>
#include <KokkosBatched_Trsm_Decl.hpp>
```

--- a/batched/dense/impl/KokkosBatched_Getrs_Serial_Impl.hpp
+++ b/batched/dense/impl/KokkosBatched_Getrs_Serial_Impl.hpp
@@ -14,10 +14,12 @@ KOKKOS_INLINE_FUNCTION static int checkGetrsInput([[maybe_unused]] const AViewTy
                                                   [[maybe_unused]] const BViewType &b) {
   static_assert(Kokkos::is_view_v<AViewType>, "KokkosBatched::getrs: AViewType is not a Kokkos::View.");
   static_assert(Kokkos::is_view_v<BViewType>, "KokkosBatched::getrs: BViewType is not a Kokkos::View.");
+  static_assert(std::is_same_v<typename BViewType::value_type, typename BViewType::non_const_value_type>,
+                "KokkosBatched::getrs: BViewType must have non-const value type.");
   static_assert(AViewType::rank == 2, "KokkosBatched::getrs: AViewType must have rank 2.");
   static_assert(BViewType::rank == 1, "KokkosBatched::getrs: BViewType must have rank 1.");
 #ifndef NDEBUG
-  const int lda = A.extent(0), n = A.extent(1);
+  const int lda = A.extent_int(0), n = A.extent_int(1);
   if (lda < Kokkos::max(1, n)) {
     Kokkos::printf(
         "KokkosBatched::getrs: the leading dimension of the array A must "
@@ -28,7 +30,7 @@ KOKKOS_INLINE_FUNCTION static int checkGetrsInput([[maybe_unused]] const AViewTy
     return 1;
   }
 
-  const int ldb = b.extent(0);
+  const int ldb = b.extent_int(0);
   if (ldb < Kokkos::max(1, n)) {
     Kokkos::printf(
         "KokkosBatched::getrs: the leading dimension of the array b must "
@@ -49,7 +51,7 @@ struct SerialGetrs<ArgTrans, Algo::Getrs::Unblocked> {
   template <typename AViewType, typename PivViewType, typename BViewType>
   KOKKOS_INLINE_FUNCTION static int invoke(const AViewType &A, const PivViewType &piv, const BViewType &b) {
     // quick return if possible
-    if (A.extent(1) == 0) return 0;
+    if (A.extent_int(1) == 0) return 0;
 
     auto info = Impl::checkGetrsInput(A, b);
     if (info) return info;

--- a/batched/dense/impl/KokkosBatched_Getrs_Serial_Internal.hpp
+++ b/batched/dense/impl/KokkosBatched_Getrs_Serial_Internal.hpp
@@ -5,6 +5,8 @@
 #define KOKKOSBATCHED_GETRS_SERIAL_INTERNAL_HPP_
 
 #include <KokkosBatched_Util.hpp>
+#include <KokkosBatched_Laswp.hpp>
+#include <KokkosBatched_Trsm_Decl.hpp>
 
 namespace KokkosBatched {
 namespace Impl {

--- a/batched/dense/src/KokkosBatched_Getrs.hpp
+++ b/batched/dense/src/KokkosBatched_Getrs.hpp
@@ -19,15 +19,17 @@ namespace KokkosBatched {
 /// \tparam BViewType: Input type for the right-hand side and the solution,
 /// needs to be a 1D view
 ///
-/// \param A [in]: A is a m by n general matrix, a rank 2 view
-/// \param piv [in]: On exit, the pivot indices, a rank 1 view
-/// \param B [inout]: right-hand side and the solution, a rank 1 view
+/// \param[in] A : A is a m by n general matrix, a rank 2 view
+/// \param[in] piv : On exit, the pivot indices, a rank 1 view
+/// \param[inout] B : right-hand side and the solution, a rank 1 view
 ///
 /// No nested parallel_for is used inside of the function.
 ///
 
 template <typename ArgTrans, typename ArgAlgo>
 struct SerialGetrs {
+  static_assert(KokkosBlas::is_trans_v<ArgTrans>, "KokkosBatched::SerialGetrs: ArgTrans must be a KokkosBlas::Trans.");
+  static_assert(std::is_same_v<ArgAlgo, Algo::Getrs::Unblocked>, "KokkosBatched::getrs: Use Algo::Getrs::Unblocked");
   template <typename AViewType, typename PivViewType, typename BViewType>
   KOKKOS_INLINE_FUNCTION static int invoke(const AViewType &A, const PivViewType &piv, const BViewType &b);
 };
@@ -35,4 +37,4 @@ struct SerialGetrs {
 
 #include "KokkosBatched_Getrs_Serial_Impl.hpp"
 
-#endif  // KOKKOSBATCHED_GETRF_HPP_
+#endif  // KOKKOSBATCHED_GETRS_HPP_

--- a/batched/dense/unit_test/Test_Batched_SerialGetrs.hpp
+++ b/batched/dense/unit_test/Test_Batched_SerialGetrs.hpp
@@ -24,7 +24,6 @@ struct Functor_BatchedSerialGetrf {
   AViewType m_a;
   PivViewType m_ipiv;
 
-  KOKKOS_INLINE_FUNCTION
   Functor_BatchedSerialGetrf(const AViewType &a, const PivViewType &ipiv) : m_a(a), m_ipiv(ipiv) {}
 
   KOKKOS_INLINE_FUNCTION
@@ -53,7 +52,6 @@ struct Functor_BatchedSerialGetrs {
   BViewType m_b;
   PivViewType m_ipiv;
 
-  KOKKOS_INLINE_FUNCTION
   Functor_BatchedSerialGetrs(const AViewType &a, const PivViewType &ipiv, const BViewType &b)
       : m_a(a), m_b(b), m_ipiv(ipiv) {}
 
@@ -89,7 +87,6 @@ struct Functor_BatchedSerialGemv {
   yViewType m_y;
   ScalarType m_alpha, m_beta;
 
-  KOKKOS_INLINE_FUNCTION
   Functor_BatchedSerialGemv(const ScalarType alpha, const AViewType &a, const xViewType &x, const ScalarType beta,
                             const yViewType &y)
       : m_a(a), m_x(x), m_y(y), m_alpha(alpha), m_beta(beta) {}
@@ -127,9 +124,9 @@ struct Functor_BatchedSerialGemv {
 ///            [1, -2]]
 ///       piv: [0, 1]
 ///
-/// \param N [in] Batch size of RHS (banded matrix can also be batched matrix)
-/// \param k [in] Number of superdiagonals or subdiagonals of matrix A
-/// \param BlkSize [in] Block size of matrix A
+/// \param[in] N  Batch size of RHS (banded matrix can also be batched matrix)
+/// \param[in] k Number of superdiagonals or subdiagonals of matrix A
+/// \param[in] BlkSize Block size of matrix A
 template <typename DeviceType, typename ScalarType, typename LayoutType, typename ParamTagType, typename AlgoTagType>
 void impl_test_batched_getrs_analytical(const int N) {
   using ats           = typename KokkosKernels::ArithTraits<ScalarType>;
@@ -185,9 +182,9 @@ void impl_test_batched_getrs_analytical(const int N) {
 
 /// \brief Implementation details of batched getrs test
 ///
-/// \param N [in] Batch size of RHS (banded matrix can also be batched matrix)
-/// \param k [in] Number of superdiagonals or subdiagonals of matrix A
-/// \param BlkSize [in] Block size of matrix A
+/// \param[in] N  Batch size of RHS (banded matrix can also be batched matrix)
+/// \param[in] k Number of superdiagonals or subdiagonals of matrix A
+/// \param[in] BlkSize Block size of matrix A
 template <typename DeviceType, typename ScalarType, typename LayoutType, typename ParamTagType, typename AlgoTagType>
 void impl_test_batched_getrs(const int N, const int BlkSize) {
   using ats           = typename KokkosKernels::ArithTraits<ScalarType>;

